templates/baremetalset/cloudinit/userdata

```diff
@@ -12,6 +12,4 @@ chpasswd:
     root:{{ .NodeRootPassword }}
   expire: False
 {{- end }}
-runcmd:
-  - dnf install -y http://download.devel.redhat.com/rcm-guest/puddles/OpenStack/rhos-release/rhos-release-latest.noarch.rpm
-  - rhos-release 16.2
\ No newline at end of file
+runcmd:
```

Comment:
Is it okay to have a key with no value here?

Reply:
runcmd is a list of commands, so I expect it to be ok to have an empty list, but we can remove it since we don't run a command with this